src/RecoveryServices/RecoveryServices/help/New-AzRecoveryServicesAsrReplicationProtectedItem.md

```diff
@@ -685,6 +744,37 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Size
+Specify the recovery virtual machine size.
+
+```yaml
+Type: System.String
+Parameter Sets: VMwareToAzureWithDiskType, VMwareToAzure, HyperVSiteToAzure
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -SqlServerLicenseType
+Specify the SQL Server license type of the VM.
+
+```yaml
+Type: System.String
+Parameter Sets: VMwareToAzureWithDiskType, VMwareToAzure, HyperVSiteToAzure
+Aliases:
+Accepted values: NoLicenseType, PAYG, AHUB
```

Comment:
>NoLicenseType [](start = 17, length = 13)

I remember 4 values in SRS

Reply:
I haven't considered the NotSpecified case because this parameter itself won't be given if the user doesn't want to specify.
Similar to LicenseType which is already present.

Reply:
Need to double check with Prachetos on how is NoLicenseType different from NotSpecified.

---
In reply to: [601548721](https://github.com/AsrOneSdk/azure-powershell/pull/21#discussion_r601548721) [](ancestors = 601548721)